src/_igraph/graphobject.c

```diff
@@ -11179,37 +11146,34 @@ PyObject *igraphmodule_Graph_cliques(igraphmodule_GraphObject * self,
     max_size = -1;
   }
 
-  if (igraph_vector_ptr_init(&result, 0)) {
+  if (igraph_vector_int_list_init(&result, 0)) {
     PyErr_SetString(PyExc_MemoryError, "");
     return NULL;
   }
 
   if (igraph_cliques(&self->g, &result, min_size, max_size)) {
-    igraph_vector_ptr_destroy(&result);
+    igraph_vector_int_list_destroy(&result);
     return igraphmodule_handle_igraph_error();
   }
 
-  n = igraph_vector_ptr_size(&result);
+  n = igraph_vector_int_list_size(&result);
   list = PyList_New(n);
   if (!list) {
     return NULL;
   }
 
   for (i = 0; i < n; i++) {
-    igraph_vector_int_t *vec = (igraph_vector_int_t *) VECTOR(result)[i];
+    igraph_vector_int_t *vec = igraph_vector_int_list_get_ptr(&result, i);
```

Comment:
This bit of code below essentially converts an `igraph_vector_int_list_t` to a Python list-of-tuples. This would be a great addition to `convert.c` and it would simplify the code in the other clique-related functions (e.g. `largest_cliques` or `maximal_cliques`). We don't need to do this as part of this PR, but if you don't, please add an issue for it so we don't forget to do this later (I can do this next week when I'm back on track).

Reply:
Added. Let me examine other places for this situation.